community-membership.md

```diff
@@ -0,0 +1,78 @@
+# Community Membership
+
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate good judgment and responsibility for the health of the subproject. The maintainer must set up the technical direction directly or by delegating these responsibilities, and make or approve design decisions for the subproject.
+
+### How to become a Maintainer:
+
+- Demonstrate a deep understanding of TiKV's technical goals and directions
+- Actively organize technical seminars and propose constructive projects
+- Can handle complex problems in the code implementation process
+- Get unanimous votes from Members of the PMC
+
+### Privileges and responsibilities:
+
+- Develop technical design and decisions for subprojects
+- Set the technical directions and priorities for subprojects
+- Participate in product releases and roadmap development of the project
+- Continuously contribute to the project by mentoring Contributors, Reviewers, and Committers
+- Ensure the overall quality of the subprojects
+
+## Committer
+
+Committers are recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A committer must have completed at least one feature or fix a major bug independently.
+
+### How to become a Committer:
+
+- Demonstrated good technical judgment
+- Assumed the Reviewer role in at least two or more subprojects
+- Reviewed or merged a certain amount of TiKV code
```

Comment:
How much exactly... I see K8s has an exact number @winkyao @gingerkidney

Reply:
It depends on different module